advanced/helm/is-with-analytics/mysql-is-with-analytics/values.yaml

```diff
@@ -0,0 +1,2036 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+mysql:
+  imageTag: "5.7"
+  mysqlRootPassword: root
+  mysqlUser: wso2carbon
+  mysqlPassword: wso2carbon
+  fullnameOverride: "wso2is-with-analytics-rdbms-service-mysql"
+  configurationFiles:
+    mysql.cnf: |-
+      [mysqld]
+      max_connections = 10000
+  initializationFiles:
+    init.sql: |-
```

Comment:
@Savidude @ThilinaManamgoda @DilanUA shall we try to break the database scripts into multiple files (in fact mapping to database scripts available in the product pack)? We may have to maintain the order of execution as follows:
- A script to create the databases and grant permissions
- Each script which creates the database tables

WDYT?

This could be considered an improvement and is not of high priority.

Reply:
+1

Reply:
+1 that sounds good